deploy/docker/Dockerfile

```diff
@@ -26,9 +27,8 @@ RUN gem install fluent-plugin-s3 -v 1.1.4 \
        && gem install fluent-plugin-prometheus -v 1.1.0 \
        && gem install fluent-plugin-kubernetes_sumologic
```

Comment:
actually, is this correct? Are we missing a version here? In fact, are we using this gem anywhere? This is https://github.com/SumoLogic/fluentd-kubernetes-sumologic, right?

Reply:
I missed that, you are right. We should include the version explicitly.  WE are using it

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl#L179

Reply:
we should use 2.4.1

Reply:
Why are we using this plugin? This is the fluend daemonset which should be removed from here as we have fluentbit now.